en/xml/fo/topic.graphic-titlepage.xml

```diff
@@ -0,0 +1,107 @@
+
+   "Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0)"
+   http://creativecommons.org/licenses/by-nc-sa/3.0/
+
+-->
+<?xml-model href="file:../5.1/dbref.rnc" type="application/relax-ng-compact-syntax"?>
+<section xml:id="dbc.fo.graphic-titlepage" remap="topic" userlevel="easy"
+    xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Graphics on Title Pages</title>
+    <subtitle>Place and format an individual graphic on a title page</subtitle>
+    <info>
+        <definitions definitionfile="defs.xml"/>
+        <keywordset>
+            <keyword>title page</keyword>
+            <keyword>cover</keyword>
+            <keyword>graphic</keyword>
+            <keyword>block</keyword>
+            <keyword>format</keyword>
+        </keywordset>
+        <subjectset>
+            <subject>
+                <subjectterm>graphic</subjectterm>
+            </subject>
+        </subjectset>
+    </info>
+
+    <section role="problem">
+        <title>Problem</title>
+        <para>You need to know how to add a graphic to your titlepage.</para>
+    </section>
+    <section role="solution">
+        <title>Solution</title>
+        <para>To add a graphic to one of your titlepages, you need to creata a custom layer for
+            titlepages as described in the <xref linkend="dbc.fo.design-titlepages"/>section.</para>
+        <para>To add your graphic, the XSL-FO specification has defined an fo:block-container to
+            encapsulate an 'external-graphics' element. Additionally, there are many many options to
+            modify your graphic on the title pages like scaling or positioning.</para>
+        <programlisting language="xml">
+&lt;fo:block space-before="1em" space-after="1em">
+    &lt;fo:external-graphic src="./images/your-graphic.svg"/>
+&lt;/fo:block>
+        </programlisting>
+        <para>Depending on the location within the customized title element, where you want to place
+            your graphic, you must place the fo:block-container before or after the title
+            block-container.</para>
+        <para>The following example will place the graphic after the document title. The graphic
+            will be scaled by 50% and the aspect ratio will be preserved.</para>
+        <programlisting language="xml">
+&lt;xsl:template match="d:title" mode="book.titlepage.recto.auto.mode">
+    &lt;fo:block xmlns:fo="http://www.w3.org/1999/XSL/Format"
+        xsl:use-attribute-sets="book.titlepage.recto.style" text-align="center"
+        font-size="32pt" space-before="18.6624pt" font-weight="bold">
+        &lt;xsl:call-template name="division.title">
+            &lt;xsl:with-param name="node" select="ancestor-or-self::d:book[1]"/>
+        &lt;/xsl:call-template>
+        &lt;fo:block space-before="1em" space-after="1em">
+            &lt;fo:external-graphic src="./images/your-graphic.svg" content-height="50%"
```

Comment:
Maybe you should replace the `images` path with something more general like the parameter `img.src.path`.

```xml
&lt;fo:external-graphic src="{$img.src.path}your-graphic.svg" content-height="50%"
```